src/librustdoc/clean/blanket_impl.rs

```diff
@@ -131,7 +131,7 @@ impl<'a, 'tcx> BlanketImplFinder<'a, 'tcx> {
                             .in_definition_order()
                             .collect::<Vec<_>>()
                             .clean(self.cx),
-                        polarity: None,
```

Comment:
This is not correct. Previously, `Some(NegativePolarity)` and `None` were treated the same.

Reply:
Oh wait ignore me - this is right, it's Positive and None that were treated the same.